docs/setup_xcode.md

```diff
@@ -234,280 +159,232 @@ $ tree
 
-If we put the user input in a file we can automate the user input.  We'll put it in a file called `main_test.in`.
+Redirect file `main_test.in` to stdin of `main.exe`.
 ```console
-$ cat main_test.in   # Peek at the contents of a file
-main_test_data.tsv
-B
-$ ./main.exe < main_test.in  # Redirect file content to main's stdin (cin)
-enter a filename
-enter a column name
-reading column B from main_test_data.tsv
-...
+$ ./main.exe < main_test.in
+What's your name?
+Hello Drew!
 ```
+{: data-highlight="1" }
 
 Without input redirection, here's how to type input in the Xcode command line.
 
 <img src="images/xcode255.png" width="768px" />
 
-Xcode no longer directly supports command-line style redirection.  One option is to simply type the input each time.  Another option is to use a work-around described in [https://gitlab.eecs.umich.edu/eecs281/wiki/wikis/xcode-file-redirection](https://gitlab.eecs.umich.edu/eecs281/wiki/wikis/xcode-file-redirection)
-
+Xcode does not support automated input redirection.  One option is to simply type the input each time.  Another option is to use a work-around described in [https://gitlab.eecs.umich.edu/eecs281/wiki/wikis/xcode-file-redirection](https://gitlab.eecs.umich.edu/eecs281/wiki/wikis/xcode-file-redirection)
 
-## Arguments and options
+### Arguments and options
 <div class="primer-spec-callout info" markdown="1">
-Skip this subsection for EECS 280 project 1.  You'll need it for project 2 and beyond.
+Skip this subsection for EECS 280 project 1.
 </div>
 
-*Arguments* and *options* are inputs to a program typed at the command line.  Arguments are often required.  Options (AKA *flags* or *switches*) start with a hyphen (`-`), and are typically optional.
-
-**Arguments example** from project 2:  `resize.exe` is the name of the program, and the arguments are `horses.ppm`,  `horses_400x250.ppm`, `400`, and `250`.
-```console
-$ ./resize.exe horses.ppm horses_400x250.ppm 400 250
-```
-{: data-variant="no-line-numbers" }
-
-**Options example** from project 5:  `main.exe` is the name of the program.  `train_small.csv` and  `test_small.csv` are arguments.  `--debug` is an option.
+Arguments and options are inputs to a program typed at the command line.  Here's an example from EECS 280 Project 5:
 ```console
 $ ./main.exe train_small.csv test_small.csv --debug
 ```
-{: data-variant="no-line-numbers" }
+{: data-variant="no-line-numbers" data-highlight="1" }
+
+- `main.exe` is the name of the program
+- `train_small.csv` and `test_small.csv` are arguments
+- `--debug` is an option
 
-To run a program with options or arguments in Xcode, select your scheme and then "Edit Scheme".
+Select your scheme, then "Edit Scheme".  You can also use menu: Product > Scheme > Edit Scheme.
 
-<img src="images/xcode258.png" width="320px" />
+| <img src="images/xcode149.png" width="384px" /> | <img src="images/xcode150.png" width="384px" /> |
 
 Add each option or argument separately.
```

Comment:
You can actually add all the arguments on the same line with spaces between, e.g. "train_small.csv test_small.csv".

IMO that's preferrable, because then you can add your different configurations (e.g. also "w16_projects_exam.csv sp16_projects_exam.csv --debug") on different lines and then toggle between them with the checkboxes.

It would be less common that you want to toggle individual arguments, as the current instructions would enable.

Reply:
Ohh, nice.  Fixed 5233304a4c93569f38882c55ecccbe57d086d8d9